pom.xml

```diff
@@ -124,43 +131,55 @@
           <systemPropertyVariables>
             <vertx.test.timeout>60</vertx.test.timeout>
             <io.netty.leakDetectionLevel>PARANOID</io.netty.leakDetectionLevel>
+            <db.host>${host}</db.host>
           </systemPropertyVariables>
           <argLine>-server -Xms128m -Xmx1024m -XX:NewRatio=2</argLine>
-          <excludes>
-            <exclude>**/ScratchPad.java</exclude>
-            <!-- Exclude Mongo tests until driver API is more stable -->
-            <exclude>**/ext/mongo/**</exclude>
-          </excludes>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>net.alchim31.maven</groupId>
-        <artifactId>scala-maven-plugin</artifactId>
-        <version>3.2.0</version>
-        <executions>
-          <execution>
-            <id>compile</id>
-            <goals>
-              <goal>compile</goal>
-            </goals>
-            <phase>compile</phase>
-          </execution>
-          <execution>
-            <id>test-compile</id>
-            <goals>
-              <goal>testCompile</goal>
-            </goals>
-            <phase>test-compile</phase>
-          </execution>
-          <execution>
-            <phase>process-resources</phase>
-            <goals>
-              <goal>compile</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>docker</id>
+      <properties>
+              <host>192.168.59.103</host>
```

Comment:
Hardcoded local IP?
I don't know how docker works and what this does, but it looks like you may want to put this into a variable...?